lib/src/default_emoji_picker_view.dart

```diff
@@ -40,51 +44,12 @@ class _DefaultEmojiPickerViewState extends State<DefaultEmojiPickerView>
   @override
   void dispose() {
     _closeSkinToneDialog();
+    _pageController.dispose();
+    _tabController.dispose();
+    _scrollController.dispose();
```

Comment:
Good catch ! I thought I was sure we are properly disposing all of them, but seems not :)